src/MLCompute/MLHelpers.cs

```diff
@@ -124,8 +179,14 @@ public static string GetDebugDescription (this MLCSoftmaxOperation self)
 		}
 	}
 
-	[iOS (14,0)][TV (14,0)][Mac (10,16)]
+#if NET
+	[SupportedOSPlatform ("ios14.0")]
+	[SupportedOSPlatform ("tvos14.0")]
+	[SupportedOSPlatform ("macos11.0")]
```

Comment:
Should this be 10.16?

Reply:
no, 10.16 is 11.0 - only the first beta using 10.16
we kind of support both (in legacy) but there's no point in moving that forward (dotnet)